plugins/content_distribution/providers/youtube_api/lib/YoutubeApiDistributionEngine.php

```diff
@@ -265,6 +269,7 @@ protected function doSubmit(KalturaDistributionSubmitJobData $data, KalturaYoutu
 			$media->setFileSize(filesize($videoPath));
 			$ingestedVideo = self::uploadInChunks($media,$videoPath, self::DEFAULT_CHUNK_SIZE_BYTE);
 			$client->setDefer(false);
+			$this->shouldIncreaseProcessedTimeout(filesize($videoPath));
```

Comment:
You call this function should which usually means it should return true or false but the function actually does change the timeout

Reply:
Changed to "setLongProcessedTimeout($fileSize);"
But it doesn't always 'set' it (depends on fileSize)
If you have a better name suggestions - I'm open to it.